src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2860,54 +2870,51 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirHumRat =
                 PartLoadRatio * state.dataWaterToAirHeatPumpSimple->LoadSideOutletHumRat +
                 (1.0 - PartLoadRatio) * state.dataWaterToAirHeatPumpSimple->LoadSideInletHumRat;
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirDBTemp =
                 PsyTdbFnHW(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirEnthalpy,
                            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirHumRat);
-            state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot = state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate;
         } else {
             // default to cycling fan, cycling compressor
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirEnthalpy = state.dataWaterToAirHeatPumpSimple->LoadSideOutletEnth;
+            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirEnthalpy = LoadSideFullOutletEnthalpy;
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirHumRat = state.dataWaterToAirHeatPumpSimple->LoadSideOutletHumRat;
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirDBTemp = state.dataWaterToAirHeatPumpSimple->LoadSideOutletDBTemp;
-            state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot = state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate * PartLoadRatio;
         }
 
         // scale heat transfer rates to PLR and power to RTF
         state.dataWaterToAirHeatPumpSimple->QLoadTotal *= PartLoadRatio;
+        state.dataWaterToAirHeatPumpSimple->QLoadTotalReport = state.dataWaterToAirHeatPumpSimple->QLoadTotal;
         state.dataWaterToAirHeatPumpSimple->QSensible *= PartLoadRatio;
         state.dataWaterToAirHeatPumpSimple->Winput *= RuntimeFrac;
-        state.dataWaterToAirHeatPumpSimple->QSource = state.dataWaterToAirHeatPumpSimple->QLoadTotal - state.dataWaterToAirHeatPumpSimple->Winput;
+        state.dataWaterToAirHeatPumpSimple->QSource =
+            state.dataWaterToAirHeatPumpSimple->QLoadTotalReport - state.dataWaterToAirHeatPumpSimple->Winput;
 
         //  Add power to global variable so power can be summed by parent object
         state.dataHVACGlobal->DXElecHeatingPower = state.dataWaterToAirHeatPumpSimple->Winput;
 
         ReportingConstant = TimeStepSys * DataGlobalConstants::SecInHour;
         // Update heat pump data structure
         state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).Power = state.dataWaterToAirHeatPumpSimple->Winput;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal = state.dataWaterToAirHeatPumpSimple->QLoadTotal;
+        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal = state.dataWaterToAirHeatPumpSimple->QLoadTotalReport;
```

Comment:
And here, QLoadTotal is reset to QLoadTotalReport. I was expecting these variables to be independent. That's why I don't see a change to the SetUpReportVariable call. But also I see a reorder of MDD and RDD files, why is that? I don't see anything that would cause that.

Reply:
They're actually independent. I know that it is confusing. The one that caused discrepancy is the `state.dataWaterToAirHeatPumpSimple->QLoadTotal` (right side of the original line 2905), while the one you said we "reset" here is `state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal`. The data structures are different between these two though they're using the same variable name. 
`state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal` is what to be reported as cooling coil cooling rate, and is used to calculate cooling coil cooling energy in [UpdateSimpleWaterToAirHP](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L3075-L3076), which I didn't notice at the first implementation.

Reply:
I don't think it is worth changing at this point.

Reply:
So what's suggested? Use `state.dataWaterToAirHeatPumpSimple->QLoadTotal` for cooling rate calculation but use `state.dataWaterToAirHeatPumpSimple->QLoadTotalReport` for cooling energy reporting?

Reply:
So now I am using `state.dataWaterToAirHeatPumpSimple->QLoadTotalReport` to report total cooling rate, total cooling energy, and also sensible and latent cooling energy to make sure they're consistent.